redash/query_runner/druid.py

```diff
@@ -48,7 +55,9 @@ def run_query(self, query, user):
         connection = connect(host=self.configuration['host'],
                              port=self.configuration['port'],
                              path='/druid/v2/sql/',
-                             scheme=self.configuration['scheme'])
+                             scheme=self.configuration['scheme'],
+                             user=self.configuration['user'],
```

Comment:
```suggestion
                             user=self.configuration.get('user'),
```

`user` might be missing from the config dict (and same for password)

Also, do you know if the client library has no side effects if passing `None` to `user`? What about an empty string?